apiary.apib

```diff
@@ -3065,79 +3506,65 @@ Returns paged collection of all Dashboards in a project.
                 "links": [
                     {
                         "rel": "self",
-                        "href": "/rest/projects/ld3y3gtm7jt6z1fx/md/dashboards?page=0&size=5&sort=id"
+                        "href": "/rest/projects/ld3y3gtm7jt6z1fx/md/dataPermissions?page=0&size=5&sort=id"
                     }
                 ],
-                "content": [
+                "contents": [
                     {
-                        "id": "j13im1r3rbvs4tus",
-                        "name": "penetration_dashboard",
-                        "type": "dashboard",
+                        "id": "b4j3pm51i1hobqfo",
```

Comment:
Tuto ukázkovou permissionu jsem si vytáhl z nosu :-) je to asi jedno, ale v dev dokumentaci používáme všechny ukázky z UK dema tak tady by bylo asi taky lepší použít https://clevermaps.atlassian.net/wiki/spaces/CDD/pages/167280805/Data+permissions